plugins/modules/image.py

```diff
@@ -125,7 +125,7 @@ def main():
         operatingsystem_id = module.lookup_entity('operatingsystem')['id']
         module.set_entity('entity', module.find_resource(
             'images',
-            search="name={0},operatingsystem={1}".format(module.foreman_params['name'], operatingsystem_id),
+            search='name="{0}",operatingsystem="{1}"'.format(module.foreman_params['name'], operatingsystem_id),
```

Comment:
Technically, we don't have to quote the operatingsystem, as it's an integer already (operatingsystem_*id*). But I don't mind too much.

Reply:
OK, for me it looks more consistent quoting both variables. If you don't mind, let's keep it.

Reply:
ACK, thanks!